### ISSUE TITLE
OC8 Server to Server with Server1 behind proxy

### ISSUE THREAD

**Issue by FTAK** [owncloud/core#14108]
Dear all,
I am trying to setup the follow:

Server-int behind proxy links a resource from server-ext using HTTP; both on OC8

Proxy has been configured in the config.php. Configuration checks message is fine in OC8.
Cent OS has been configured with HTTP_PROXY environment variables.
yum, curl, etc. in shell working fine but not oc8 with curl in php (5.4.37)

Error message from owncloud.log
```json
{"reqId":"f8305f404b72798d288af5811fd0ac12",
"remoteAddr":"<ip>",
"app":"files_external",
"message":"Exception: {\"Message\":\"Sabre\\\\DAV\\\\Exception: [CURL] Error while making request: couldn't connect to host (error code: 7)\",\"Code\":0,\"Trace\":\"#0 \\\/var\\\/www\\\/html\\\/owncloud\\\/lib\\\/private\\\/files\\\/storage\\\/dav.php(382): OC\\\\Files\\\\Storage\\\\DAV->convertSabreException(Object(Sabre\\\\DAV\\\\Exception))\\n#1 \\\/var\\\/www\\\/html\\\/owncloud\\\/lib\\\/private\\\/files\\\/storage\\\/common.php(393): OC\\\\Files\\\\Storage\\\\DAV->stat('')\\n#2 \\\/var\\\/www\\\/html\\\/owncloud\\\/apps\\\/files_external\\\/lib\\\/config.php(457): OC\\\\Files\\\\Storage\\\\Common->test(true)\\n#3 \\\/var\\\/www\\\/html\\\/owncloud\\\/apps\\\/files_external\\\/lib\\\/config.php(433): OC_Mount_Config::getBackendStatus('\\\\\\\\OC\\\\\\\\Files\\\\\\\\Stora...', Array, true)\\n#4 \\\/var\\\/www\\\/html\\\/owncloud\\\/apps\\\/files_external\\\/personal.php(29): OC_Mount_Config::getPersonalMountPoints()\\n#5 \\\/var\\\/www\\\/html\\\/owncloud\\\/lib\\\/private\\\/app.php(713): include('\\\/var\\\/www\\\/html\\\/o...')\\n#6 \\\/var\\\/www\\\/html\\\/owncloud\\\/settings\\\/personal.php(108): OC_App::getForms('personal')\\n#7 \\\/var\\\/www\\\/html\\\/owncloud\\\/lib\\\/private\\\/route\\\/route.php(135) : runtime-created function(1): require_once('\\\/var\\\/www\\\/html\\\/o...')\\n#8 [internal function]: __lambda_func(Array)\\n#9 \\\/var\\\/www\\\/html\\\/owncloud\\\/lib\\\/private\\\/route\\\/router.php(250): call_user_func('\\\\x00lambda_353', Array)\\n#10 \\\/var\\\/www\\\/html\\\/owncloud\\\/lib\\\/base.php(763): OC\\\\Route\\\\Router->match('\\\/settings\\\/perso...')\\n#11 \\\/var\\\/www\\\/html\\\/owncloud\\\/index.php(36): OC::handleRequest()\\n#12 {main}\",\"File\":\"\\\/var\\\/www\\\/html\\\/owncloud\\\/lib\\\/private\\\/files\\\/storage\\\/dav.php\",\"Line\":594}",
"level":4,
"time":"2015-02-11T14:25:56+00:00"}
```

By reading some older posts, it's said, that owncloud does not have proxy support for outbound?!

Thank you very much in advance.

Fabian.

**Comment by karlitschek** [comment]
@schiesbn

**Comment by PVince81** [comment]
That might be more of a curl problem ?

@LukasReschke @icewind1991

**Comment by PVince81** [comment]
Proxy support should be fixed in 8.1 through https://github.com/owncloud/core/pull/15195.

Please try again with that version and reopen if the issue persists. Thanks.
